deepchecks/checks/performance/calibration_score.py

```diff
@@ -49,15 +49,15 @@ def _calibration_score(self, dataset: Dataset, model):
         y_pred = model.predict_proba(ds_x)
 
         briers_scores = {}
-        unique_labels = dataset.label_col.unique()
+        unique_labels = model.classes_
```

Comment:
What is the difference between classes_ and label_col.unique()?

Reply:
exactly what you said in the comment